cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperator.java

```diff
@@ -130,7 +130,7 @@ public void start(Promise<Void> start) {
         Future.join(startFutures)
                 .compose(f -> {
                     LOGGER.info("Setting up periodic reconciliation for namespace {}", namespace);
-                    this.reconcileTimer = vertx.setPeriodic(this.config.getReconciliationIntervalMs(), res2 -> {
+                    this.reconcileTimer = vertx.setPeriodic(this.config.getReconciliationIntervalMs(), timerId -> {
```

Comment:
This seems unused. So should it be names `ignored` instead?

Reply:
Yeah, ignored is better in this case. Thanks